scss/utilities/_spacing.scss

```diff
@@ -14,18 +14,18 @@
     $length-x:   map-get($lengths, x);
     $length-y:   map-get($lengths, y);
 
-    .#{$abbrev}-a-#{$size} { #{$prop}:        $length-y $length-x !important; } // a = All sides
-    .#{$abbrev}-t-#{$size} { #{$prop}-top:    $length-y !important; }
-    .#{$abbrev}-r-#{$size} { #{$prop}-right:  $length-x !important; }
-    .#{$abbrev}-b-#{$size} { #{$prop}-bottom: $length-y !important; }
-    .#{$abbrev}-l-#{$size} { #{$prop}-left:   $length-x !important; }
+    .#{$abbrev}-#{$size} { #{$prop}:        $length-y $length-x !important; } // a = All sides
```

Comment:
Nope. Think of it like the CSS properties—`margin-left` goes with `m-l` while `margin` goes with `m`.